src/ipc/dma-copy.c

```diff
@@ -184,8 +184,6 @@ int dma_copy_new(struct dma_copy *dc)
 	return 0;
 }
 
-#if CONFIG_DMA_GW
-
```

Comment:
this config is not AMD specific, if AMD platform want to use this, better to add this config to AMD build?

Reply:
Enabling CONFIG_DMA_GW will bring in many other API's which will cause failures in DMA operations  and are not required on AMD platform, but dma_copy_set_stream_tag is required for enabling DMA channel based on stream tag, That  is the reason to remove CONFIG_DMA_GW for this API, I believe this won't affect non AMD platforms.

Reply:
yeah, this is fine - its mostly generic if anyone is using a DMA tag to identify a DMA channel.